Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -51,7 +55,7 @@ public ApplicationContextManagerBlazor(IHttpContextAccessor hca, AuthenticationS
     /// <summary>
     /// Gets or sets a reference to the current ApplicationContext.
     /// </summary>
-    public ApplicationContext ApplicationContext { get; set; }
+    public ApplicationContext? ApplicationContext { get; set; }
```

Comment:
Shouldn't the setter throw if a null is provided like in the other context manager?

Reply:
We don't have any context manager which guards against null.
I'd love to make sure the `ApplicationContext` is always set. But I think that's not possible due to possible circular references with `ApplicationContextAccessor`?
Or we make the `set` internal and say "As long as you use the framework as it's intended we guarantee it's not null"? 🤔